File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md

```diff
@@ -340,6 +340,285 @@ using (WordDocument document = new WordDocument())
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Footnotes-and-Endnotes/Add-endnotes-in-Word-document).
 
+## Adding a Footnote and Endnote position
+
+Footnotes are typically located at the bottom of a page or beneath the text being referenced, and endnotes are typically placed at the end of a document or at the end of a section. This can be done using `FootnotePosition` API and `EndnotePosition` API.
+
+The following code example illustrates how to add positions of footnotes and endnotes:
+
+{% tabs %}  
+
+{% highlight c# tabtitle="C#" %}
+//Create a new Word document.
+using (WordDocument document = new WordDocument())
+{
+//Create a section.
```

Comment:
@sharonaumao - Please align the code snippets properly as similar to below UG topic,
https://github.com/syncfusion-content/fileformat-docs/edit/master/File-Formats/DocIO/mail-merge/mail-merge-options.md